.circleci/set-up-globals.sh

```diff
@@ -30,3 +30,11 @@ git config --global user.email "$GIT_EMAIL"
 git config --global user.name "Circle CI"
 # Ignore file permissions.
 git config --global core.fileMode false
+
+# Get public key and add to Pantheon
+cd ~/.ssh
+openssl rsa -in id_rsa -out id_rsa_ssh2.pub -pubout
+cat id_rsa_ssh2.pub
+ssh-keygen -f id_rsa_ssh2.pub -i -m pkcs8 > id_rsa.pub
+chmod 0600 id_rsa.pub
+terminus ssh-key:add id_rsa.pub
```

Comment:
Why is a new ssh key being added here each test?

Note that if you keep adding ssh keys and never remove them, they will pile up in the dashboard. Once you get somewhere > 200 of them, all of them stop working until a sufficient number are deleted.

Reply:
[Cleanup fixtures](https://github.com/pantheon-systems/terminus-build-tools-plugin/blob/master/.circleci/cleanup-fixtures.sh) removes excess keys if they match a specific pattern.